doc/api_2_0_spec/api_2_0_spec.md

```diff
@@ -771,6 +771,33 @@ the returned data would look like:
 }
 ```
 
+## api/v2/stories_public/is_syndicated_ap (POST)
+
+Detect whether a given block of content is likely to be ap syndicated content by looking for certain signals in the text
+(for example 'boston (ap)' and by comparing the text to the text of ap content in the Media Cloud database.
+
+### Query Parameters
+
+| Parameter          | Default          | Notes
+| ------------------ | ---------------- | ----------------------------------------------------------------
+| `content`          | n/a              | text or html content
+
+### Output Description
+
+| Field                        | Description
+| ---------------------------- | -----------------------------------------------------------------------------
+| is_syndicated                | 1 if the story is syndicated, 0 otherwise
+
+### Example
+
+URL: https://api.mediacloud.org/api/v2/stories_public/is_syndicated_ap?content=foo
```

Comment:
You probably want to `POST` the story content (as a JSON?) instead of having it as a `GET` query parameter here because HTTP URLs are limited in size (the limit depends on the implementation, in most browsers it's around 2000 bytes).

Reply:
fixed.  I changed the call to a PUT, which I guess makes sense because this is an idempotent operation (though it doesn't change anything, so it doesn't really fit).